packages/app-form-builder/src/admin/components/FormEditor/Context/functions/moveStep.ts

```diff
@@ -1,24 +1,25 @@
-import { FbFormStep } from "~/types";
+import { FbFormStep, MoveStepParams } from "~/types";
 
-interface MoveStepParams {
+interface MoveStep extends MoveStepParams {
     data: FbFormStep[];
-    /* formStep is the step that we are dragging */
-    formStep: FbFormStep;
-    step: FbFormStep;
 }
 
-const moveStep = (params: MoveStepParams) => {
-    const { step, data, formStep } = params;
+const moveStep = (params: MoveStep) => {
+    const { target, destination, data } = params;
 
-    /* step1 is the step that will change it's position with */
-    const step1 = data.findIndex((v: FbFormStep) => v.id === step.id);
-    /* step2 is the step that is being dragged */
-    const step2 = data.findIndex((v: FbFormStep) => v.id === formStep.id);
+    /* targetStep is the step that is being dragged */
```

Comment:
```suggestion
// `targetStep` is the step that is being dragged.
```

Please use `//` for these one line comments.